Tests/TestingApps/PurchaseTesterSwiftUI/PurchaseTester.xcodeproj/project.pbxproj

```diff
@@ -675,30 +681,36 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CLANG_MODULES_AUTOLINK = YES;
 				CODE_SIGN_ENTITLEMENTS = PurchaseTester.entitlements;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Manual;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = 8SXR2327BM;
 				ENABLE_PREVIEWS = YES;
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_FILE = "PurchaseTester-Info.plist";
 				INFOPLIST_KEY_CFBundleDisplayName = "Purchase Tester";
+				INFOPLIST_KEY_LSApplicationCategoryType = "public.app-category.developer-tools";
 				INFOPLIST_KEY_UIApplicationSceneManifest_Generation = YES;
 				INFOPLIST_KEY_UIApplicationSupportsIndirectInputEvents = YES;
 				INFOPLIST_KEY_UILaunchScreen_Generation = YES;
 				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPad = "UIInterfaceOrientationPortrait UIInterfaceOrientationPortraitUpsideDown UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight";
 				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPhone = "UIInterfaceOrientationPortrait UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight";
-				LD_RUNPATH_SEARCH_PATHS = (
-					"$(inherited)",
-					"@executable_path/Frameworks",
-				);
+				LD_RUNPATH_SEARCH_PATHS = "@executable_path/Frameworks";
+				"LD_RUNPATH_SEARCH_PATHS[sdk=macosx*]" = "@executable_path/../Frameworks";
```

Comment:
These crashes:
> Termination Reason:    Namespace DYLD, Code 1 Library missing
Library not loaded: @rpath/RevenueCat_76EF69325FE509FC_PackageProduct.framework/Versions/A/RevenueCat_76EF69325FE509FC_PackageProduct
Referenced from: <DEACE43D-F7AB-307B-A35E-44EC03198E29> /Applications/PurchaseTester.app/Contents/MacOS/PurchaseTester
Reason: tried: '/Applications/PurchaseTester.app/Contents/MacOS/Frameworks/RevenueCat_76EF69325FE509FC_PackageProduct.framework/Versions/A/RevenueCat_76EF69325FE509FC_PackageProduct' (no such file), '/Applications/PurchaseTester.app/Contents/MacOS/Frameworks/RevenueCat_76EF69325FE509FC_PackageProduct.framework/Versions/A/RevenueCat_76EF69325FE509FC_PackageProduct' (no such file), '/System/Volumes/Preboot/Cryptexes/OS@rpath/RevenueCat_76EF69325FE509FC_PackageProduct.framework/Versions/A/RevenueCat_76EF69325FE509FC_PackageProduct' (no such file), '/Applications/PurchaseTester.app/Contents/MacOS/Frameworks/RevenueCat_76EF69325FE509FC_PackageProduct.framework/Versions/A/RevenueCat_76EF69325FE509FC_PackageProduct' (no such file), '/Applications/PurchaseTester.a

Were due to this being wrong. `macOS` has a slightly different folder structure.
@aboedo this was the solution.